src/server/main.py

```diff
@@ -21,7 +21,7 @@
 async def index(request: Request) -> Response:
     """TODO: this is the main hompage, should have a bubble map which should
     link ot the /model"""
-    bubble_map_script = server_document(os.path.join(BOKEH_URI, "app2"))
+    bubble_map_script = server_document(BOKEH_URI + "/app2")
```

Comment:
```suggestion
    bubble_map_script = server_document(f"{BOKEH_URI}/app2")
```

Nit pick. See app1... Just to have it the same way.